### PR TITLE
Bump core to v0.11

### PR DESCRIPTION
**Pull request by FelonEkonom**

### CODE REVIEW

File: test/depayloader_pipeline_test.exs [hunk 1]
```diff
@@ -10,31 +10,32 @@ defmodule Membrane.RTP.H264.DepayloaderPipelineTest do
...
       STAPFactory.sample_data()
       |> Enum.each(fn elem ->
         assert_sink_buffer(pid, :sink, buffer)
         assert %Buffer{payload: payload} = buffer
-        assert <<1::32, elem::binary()>> == payload
+        assert <<1::32, elem::binary>> == payload
       end)
 
-      Membrane.Pipeline.stop_and_terminate(pid, blocking?: true)
+      Membrane.Pipeline.terminate(pid, blocking?: true)
```
Comment (mickel8) [inline test/depayloader_pipeline_test.exs:31]: Is this needed? You start pipeline using `start_link_supervised`

File: test/depayloader_pipeline_test.exs [hunk 2]
```diff
@@ -10,31 +10,32 @@ defmodule Membrane.RTP.H264.DepayloaderPipelineTest do
...
+      {:ok, _supervisor_pid, pid} =
         STAPFactory.sample_data()
         |> Enum.chunk_every(2)
         |> Enum.map(&STAPFactory.into_stap_unit/1)
         |> Enum.map(&%Membrane.Buffer{payload: &1})
         |> Source.output_from_buffers()
         |> DepayloaderTestingPipeline.start_pipeline()
 
-      Membrane.Pipeline.play(pid)
+      # Membrane.Pipeline.play(pid)
```
Comment (mickel8) [inline test/depayloader_pipeline_test.exs:21]: Leftover, please check other places too

File: test/support/depayloader_testing_pipeline.ex [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
-      ]
-    }
+    structure = [
+      child(:source, %Testing.Source{output: data, stream_format: %RTP{}})
+      |> child(:depayloader, Depayloader)
+      |> child(:sink, Testing.Sink)
+    ]
 
-    Pipeline.start_link(options)
+    Pipeline.start_link_supervised(structure: structure)
```
Comment (mickel8) [inline test/support/depayloader_testing_pipeline.ex:18]: I would favor the version with `!`

File: lib/rtp_h264/depayloader.ex [hunk 4]
```diff
@@ -30,38 +30,38 @@ defmodule Membrane.RTP.H264.Depayloader do
...
+    {[], state}
   end
 
   @impl true
   def handle_process(:input, buffer, _ctx, state) do
     with {:ok, {header, _payload} = nal} <- NAL.Header.parse_unit_header(buffer.payload),
          unit_type = NAL.Header.decode_type(header),
-         {{:ok, actions}, state} <- handle_unit_type(unit_type, nal, buffer, state) do
-      {{:ok, actions}, state}
+         {actions, state} when is_list(actions) <- handle_unit_type(unit_type, nal, buffer, state) do
```
Comment (daniel-jodlos) [inline lib/rtp_h264/depayloader.ex:53]: I *really* don't like that guard. In fact, I think I would keep the return of `handle_unit_type` as it was. It's not a Membrane callback and it definitely needs to be able to return an error. A standard way to do it is `{:ok, something} | {:error, reason}` and I would stick to that.

File: test/depayloader_pipeline_test.exs (hunk 1 above)
Comment (daniel-jodlos) [reply test/depayloader_pipeline_test.exs:31]: I think it's required. `start_link` doesn't act when the process terminates cleanly.

File: test/depayloader_pipeline_test.exs [hunk 5]
```diff
@@ -45,14 +46,15 @@ defmodule Membrane.RTP.H264.DepayloaderPipelineTest do
...
```
Comment (daniel-jodlos) [inline test/depayloader_pipeline_test.exs:49]: Another leftover

**Review by daniel-jodlos — CHANGES_REQUESTED** [review]

**Review by mickel8 — APPROVED** [review]

**Pull request merged**